src/celeritas/em/msc/detail/UrbanMscScatter.hh

```diff
@@ -271,8 +274,6 @@ CELER_FUNCTION auto UrbanMscScatter::operator()(Engine& rng) -> MscInteraction
         if (ipow<2>(theta0_) < shared_.params.tau_small)
         {
             // Very small outgoing angular distribution
-            // XXX why are we comparing the square of an angle to an MFP
-            // value??
```

Comment:
Whoops didn't mean to remove this comment... I get why you'd want to return an exact forward scatter for small outgoing angle but I don't get why the comparison is against `tau_small`—perhaps it's another arbitrary constant that has the same value?